defaults/test_default.yaml

```diff
@@ -13,6 +13,7 @@ manager_scylla_backend_version: '2022'
 experimental: true
 round_robin: false
 
+reactor_stall_tolerable_threshold: 500
```

Comment:
So what was the value before that?

Reply:
the value was `500`, but hardcoded in the `sdcm/sct_events/database.py` file

BTW, i missed changing it in there, and now it is fixed